mesh/nurbs.hpp

```diff
@@ -403,6 +405,21 @@ class NURBSExtension
    void UniformRefinement();
    void KnotInsert(Array<KnotVector *> &kv);
    void KnotInsert(Array<Vector *> &kv);
+
+   int GetElementPatch(int elem) const { return el_to_patch[elem]; }
+   void GetElementIJK(int elem, Array<int> & ijk);
+
+   const Array3D<int>& GetPatchDofs(const int patch);
+
+   int NumPatchDofs1D(const int patch, const int dim);
+
+private:
+   std::vector<Array3D<int>> patchDofs;
```

Comment:
`mfem::Vector` is only of type `double`, so I think what you mean is to use `mfem::Array`. It can only take trivial types, so `Array<Array3D<int>>` is not possible.

Reply:
Clear!